boards/atxmega-a1u-xpro/include/openwsn_defs.h

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright (C) 2020 Inria
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     boards_atxmega-a1u-xpro
+ * @{
+ *
+ * @file
+ * @brief       Board specific definitions for pkg/openwsn
+ *
+ * @author      Francisco Molina <francois-xavier.molina@inria.fr>
+ */
+
+#ifndef OPENWSN_DEFS_H
+#define OPENWSN_DEFS_H
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @name    OpenWSN timing constants
+ *
+ * @{
+ */
+/* Measured 800us + ~10% */
+#define PORT_maxTxDataPrepare   (900/PORT_US_PER_TICK)
+/* Measured 450us + ~10% */
+#define PORT_maxRxAckPrepare    (500/PORT_US_PER_TICK)
+/* Measured 500us + ~10% */
+#define PORT_maxRxDataPrepare   (550/PORT_US_PER_TICK)
+/* Measured 660us + ~10% */
+#define PORT_maxTxAckPrepare    (750/PORT_US_PER_TICK)
+/* Measured 650us with openwsn_sctimer_rtt */
+#define PORT_delayTx            (650/PORT_US_PER_TICK)
```

Comment:
Care to split this out? I have not worked with OpenWSN so far. I think @fjmolinas will only need to take a quick look to review these.

I have the feeling that these parameters depend on the specific transceiver used, right? Since the AT85RF231 is plugged in as separate break out board, other pin compatible break out boards (e.g. all the other AT86RF* seem to also be available as RZ-600 break out) could also be used. Will those constants still apply?